### PR TITLE
create models

### PR DESCRIPTION
**Pull request by marianaWest**
I created three Mongoose schemas: User (copied from Leon's template), BirthdayPerson (the only name that came up), and Gift. Please take a look. Also, I wasn't sure how to add the birthday person's id to the gift schema. Any suggestions?

### CODE REVIEW

**Review by emtek995 — CHANGES_REQUESTED** [review]
The type that mongoose uses for the IDs in the database is ObjectId.
You can use 
birthdayPerson: { type: ObjectId, .......

**Comment by marianaWest** [comment]
thank you Edward, I'll make the change.

File: models/BirthdayPerson.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    required: true,
+  },
+  relation: {
+    type: String,
+    required: false,
+  },
+  userId: {
+    type: String,
+    required: true,
+  }
```
Comment (emtek995) [inline models/BirthdayPerson.js:19]: giftId: {
    type: String,
    required: true,
  }

**Review by emtek995 — APPROVED** [review]
Looks really good.

**Pull request merged**